go.mod

```diff
@@ -4,86 +4,84 @@ go 1.15
 
 require (
 	github.com/akutz/gofsutil v0.1.2
-	github.com/container-storage-interface/spec v1.2.0
+	github.com/container-storage-interface/spec v1.3.0
 	github.com/coreos/etcd v3.3.25+incompatible // indirect
 	github.com/davecgh/go-spew v1.1.1
 	github.com/docker/spdystream v0.0.0-20181023171402-6480d4af844c // indirect
 	github.com/elazarl/goproxy v0.0.0-20200710112657-153946a5f232 // indirect
 	github.com/elazarl/goproxy/ext v0.0.0-20200710112657-153946a5f232 // indirect
 	github.com/fsnotify/fsnotify v1.4.9
-	github.com/golang/protobuf v1.5.1
+	github.com/go-kit/kit v0.10.0 // indirect
+	github.com/golang/protobuf v1.5.2
 	github.com/google/uuid v1.2.0
-	github.com/googleapis/gnostic v0.5.4 // indirect
 	github.com/grpc-ecosystem/go-grpc-middleware v1.2.0 // indirect
 	github.com/grpc-ecosystem/grpc-gateway v1.14.5 // indirect
 	github.com/kr/text v0.2.0 // indirect
 	github.com/kubernetes-csi/csi-lib-utils v0.7.0
 	github.com/niemeyer/pretty v0.0.0-20200227124842-a10e7caefd8e // indirect
-	github.com/onsi/ginkgo v1.16.2
-	github.com/onsi/gomega v1.10.5
+	github.com/onsi/ginkgo v1.16.4
+	github.com/onsi/gomega v1.13.0
```

Comment:
make sure e2e tests are not affected with gomega bump up.

Reply:
Same reason as above.
I have run the pre-checkin e2e tests for different flavors and they were successful. So if tests get picked from the PR, then we will be good.
But I will ask someone from FVT team to review it as well for test framework and e2e updates.